tutorpod_autoscaling/templates/pod-autoscaling/k8s/keda.yml

```diff
@@ -0,0 +1,23 @@
+{% for worker, autoscaling in POD_AUTOSCALING_KEDA_CONFIG.items() %}
+---
+apiVersion: keda.sh/v1alpha1
+kind: Keda
+metadata:
+  name: {{ worker }}-scaledobject
+spec:
+  minReplicaCount: {{ autoscaling["min_replicas"] }}
+  maxReplicaCount: {{ autoscaling["max_replicas"] }}
+  scaleTargetRef:
+    kind: Deployment
+    name: {{ worker }}
+  triggers:
+  - metadata:
```

Comment:
Could you please explain how this scaling based on Redis works?

Reply:
It reads the Redis list and based on its length it creates a pod per `list_length` till its max replicas.